### PR TITLE
Added new enum for Props and new Methods for Entity class to find closest Vehicles, Peds, Props

### PR DESCRIPTION
**Pull request by manups4e**

### CODE REVIEW

**Comment by manups4e** [comment]
i don't get why is the FXServer = 1 build not building correctly 😅 i actually only edited Client API

**Comment by Local9** [comment]
This will be very handy, will be much nicer to have a central version instead of the horrors I've implemented.

**Comment by manups4e** [comment]
i'd like to get a review about this :D

File: code/client/clrcore/External/PropHash.cs [hunk 1]
```diff
@@ -0,0 +1,11456 @@
...
+		ss1_rd1_19 = -1907152414,
+		ss1_rd1_20 = 817229101,
+		ss1_rd1_21 = 520636882,
+		ss1_rd1_22 = 319369684,
+		ss1_rd1_24 = 2041413403,
+		ss1_rd1_25 = 1743903652,
+		ss1_rd1_26 = -546026801,
+		ss1_rd1_27 = -248844740,
+		ss1_rd1_28 = 2077754264,
+		ss1_rd1_29 = -19
```
Comment (Disquse) [inline code/client/clrcore/External/PropHash.cs:9188]: Are you sure those are *prop* model hashes? There's lots of ped/vehicle model hashes.

Comment (manups4e) [reply code/client/clrcore/External/PropHash.cs:9188]: yes as most vehicles can be spawn as object (not interactable)

**Comment by blattersturm** [comment]
Useful as reference for future C# API improvements.

**Pull request closed without merging**